types/underscore/index.d.ts

```diff
@@ -802,26 +801,12 @@ declare module _ {
         /**
         * @see _.first
         **/
-        head<T>(array: _.List<T>): T | undefined;
+        head: UnderscoreStatic['first'];
 
         /**
         * @see _.first
         **/
-        head<T>(
-            array: _.List<T>,
-            n: number): T[];
-
-        /**
-        * @see _.first
-        **/
-        take<T>(array: _.List<T>): T;
-
-        /**
-        * @see _.first
-        **/
-        take<T>(
-            array: _.List<T>,
-            n: number): T[];
+        take: UnderscoreStatic['first'];
```

Comment:
I had originally updated all of the functions definitions for those that had aliases as I found inconsistencies (such as the return types of  `first<T>(array: _.List<T>): T | undefined;` and its alias `take<T>(array: _.List<T>): T;` being inconsistent) but then realized that I could do this instead. How do folks feel about doing this in general? It's nice because it helps prevent inconsistencies and reduces the edits required to update a function with aliases, but on the other hand it's a little weird because this then ends up being shown as a property instead of a method in some IDEs.

I originally wrote tests that targeted each alias as well as the primary function for groups like these, but if folks are ok with this I could probably drop the tests targeting aliases to help reduce the size of the tests file and reduce the number of changes that need to be made in test files when these definitions require an update.

Reply:
Per [this feedback](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45201#issuecomment-637479708), leaving both this pattern for aliasing and tests for aliases in place.